### DISCUSSION TITLE
我新加坡的机子偷cloudflare.net的可以吗？

### DISCUSSION THREAD

**Discussion by osongsongo** [XTLS/Xray-core#2316] [Q&A]
也支持h2，好像也符合标准。或者新加坡有更好的网站可以选择吗？偷自己我还不会用。

**Comment by chika0801** [comment]
没固定标准，我的各地区VPS调试时，我都用 https://www.lovelive-anime.jp/ 也没事。只要你本地ping你dest填的网址不要返回CN IP这个起码的意识你有就是了。

**Reply by osongsongo** [reply]
我用NGINX开启h2跟tls1.3，然后dest直接填自己的域名就是偷自己了吗？还需要设置其他东西吗？

**Reply by chika0801** [reply]
我在dest那儿没用过填自己域名，我在交流时说偷自己的形式，我是指Dest那填本机的一个端口号比如8003，这里是省略了127.0.0.1:8003。然后你安装的nginx配置中是监听本机 127.0.0.1 8003 这样2个程序Xray和Nginx来互相通讯的。Nginx配置中加载SSL证书。

配置示例参考 https://github.com/chika0801/Xray-examples/tree/main/VLESS-XTLS-uTLS-REALITY/steal_yourself

**Reply by osongsongo** [reply]
好麻烦，还是用CF的算了😢😢

**Reply by chika0801** [reply]
学自建，就是入坑到XX的路。你把这方面当兴趣可能会好点。看个人想不想花时间精力学了。

**Reply by osongsongo** [reply]
貌似xui是没有禁用回国流量的吧，现在可以设置吗？还是得去配置文件里改？

**Reply by chika0801** [reply]
我不用XUI，不清楚它默认配置是怎么样的。我记得它要地方可以让你加。我是手动安装手动改配置文件派。

**Reply by chika0801** [reply]
另外，建议你加XRAY的TG群，找群友交流

**Reply by MakeFortuneQuietly** [reply]
还是偷银行/证券比较好？墙虽然连Github都不发放过，不会发疯到自毁经济吧。

https://www.nomura.co.jp/
https://www.amundi.lu/
https://www.ing.nl/

**Reply by Smallthing** [reply]
鉴定为洗钱

**Comment by FranzKafkaYu** [comment]
> 貌似xui是没有禁用回国流量的吧，现在可以设置吗？还是得去配置文件里改？

在web后台修改配置模板即可。

**Comment by spldmanager** [comment]
我有个疑问，能不能直接就偷vps对应的服务商的网站呢，比方说甲骨文的就偷甲骨文的官网。这样子ip最接近，也更会被认为就是官网的cdn
